javascript-modal/styles.css

```diff
@@ -0,0 +1,46 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  width: 1200px;
+  height: 100vh;
+}
+
+button {
+  color: white;
+  font-size: 1.2rem;
+  padding: 5px 20px;
+  border: none;
+  border-radius: 5px;
+}
+
+button.open {
+  background-color: rgb(76, 76, 219);
+}
+
+button.survey {
+  background-color: tomato;
+  width: 20%;
+  justify-content: center;
+  box-shadow: 0 2px 7px 1px rgba(0, 0, 0, 0.64);
+}
+
+div.modal {
```

Comment:
The name for this class is somewhat misleading, since not showing the modal is named "modal". This class would be better named something like "hidden"